BackEnd/src/main/java/com/unosquare/admin_core/back_end/service/EventService.java

```diff
@@ -73,17 +69,17 @@ public void saveEvents(EventDTO... eventDtos) {
 
     @Transactional
     public void updateEvent(UpdateEventDTO updateEventDTO) {
-        Preconditions.checkNotNull(updateEventDTO);
+    /*    Preconditions.checkNotNull(updateEventDTO);
         Optional<Event> retrievedEvent = eventRepository.findById(updateEventDTO.getEventId());
         if (retrievedEvent.isPresent()) {
             Event event = retrievedEvent.get();
             modelMapper.map(updateEventDTO, event);
             if (updateEventDTO.getMessage() != null) {
-                EventMessage eventMessage = mapToEventMessage(updateEventDTO.getMessage(), event, event.getEmployee(), EventMessageTypes.GENERAL.getEventStatusId());
-                saveEventMessage(eventMessage);
+               EventMessage eventMessage = mapToEventMessage(updateEventDTO.getMessage(), event, event.getEmployee(), EventMessageTypes.GENERAL.getEventStatusId());
+               saveEventMessage(eventMessage);
             }
             save(event);
-        }
+        }*/
```

Comment:
Why this commented out and not removed?

Reply:
updateHoliday() in HolidayController is the only place that this method is called but updateHoliday itself is never called. Infact no methods ni HolidayController are called, is this a no longer used class?

Reply:
These expose the endpoints at /holiday/

Reply:
This PR is still a work in progress, it looks like Eoin hadn't got near the update yet.